apps/mobile/app/utils/crashReporting.ts

```diff
@@ -23,46 +23,45 @@
  *  This is where you put your crash reporting service initialization code to call in `./app/app.tsx`
  */
 export const initCrashReporting = () => {
-  // Sentry.init({
-  //   dsn: "YOUR DSN HERE",
-  //   enableInExpoDevelopment: true,
-  //   debug: true, // If `true`, Sentry will try to print out useful debugging information if something goes wrong with sending the event. Set it to `false` in production
-  // })
-  // Bugsnag.start("YOUR API KEY")
+	Sentry.init({
```

Comment:
Please add to the code above check if `process.env.EXPO_PUBLIC_SENTRY_DSN` is defined and only call .init method if it is. Otherwise skip all initialization/usage of Sentry! I.e. if we did not configure it, it should not cause app crash etc.